health/notifications/alarm-notify.sh.in

```diff
@@ -428,6 +428,10 @@ else
   done
 fi
 
+if [[ ! $curl_options =~ .*\--connect-timeout ]]; then
+  curl_options+=" --connect-timeout 5"
+fi
```

Comment:
This is right after reading(loading) configuration files. `curl_options` is defined in there and is an empty string by default.

https://github.com/netdata/netdata/blob/c3dba899af944853aaf2f4be0f7a3be24ae0e29d/health/notifications/health_alarm_notify.conf#L127-L132

Reply:
Hi @ilyam8 

Wouldn't we expect that if someone edits the `health_alarm_notify.conf` file to pass `--connect-timeout` with a number he wishes?

Reply:
Hi, @ilyam8. I don't understand your question, sorry. Can you reword it?

Reply:
Sorry sorry, you explained, mis-read.

Reply:
Great. Can I get a review? If you have a better suggestion on how to solve the problem, please share. If you think that we need to keep the logic as it is, please provide your reasoning. Thanks!